src/main/java/it/gov/pagopa/idpay/transactions/exception/NotEnoughFiltersException.java

```diff
@@ -0,0 +1,7 @@
+package it.gov.pagopa.idpay.transactions.exception;
+
+public class NotEnoughFiltersException extends RuntimeException{
```

Comment:
don't create it as a new Exception. we should do this only if it's a recurring exception.
You can delete this